packages/loot-core/src/server/schedules/app.ts

```diff
@@ -65,7 +65,7 @@ export function updateConditions(conditions, newConditions) {
   return updated.concat(added);
 }
 
-export function getNextDate(dateCond, start = new Date()) {
+export function getNextDate(dateCond, start = new Date(currentDay())) {
```

Comment:
This is for the schedules table to show "due" instead of "scheduled" when using tests.